src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -82,6 +124,10 @@ pushd $EXECUTION_DIR
 test_exitcode=$?
 popd
 echo ----- end $(date +"%T") ----- exit code $test_exitcode ----------------------------------------------------------
+
+if [[ ($test_exitcode -ge 129 && $test_exitcode -le 160) || $test_exitcode -eq 0 ]]; then
```

Comment:
By doing this you have put knowledge here of the contents of the array, so if it is changed, someone has to find this and fix it as well.
Better to just do something like this
```
if [ "${exitcode_list[$test_exitcode]}" -ne ""] 
     echo exit code $test_exitcode means ${exitcode_list[$test_exitcode]}
fi
```
I may have the syntax not quite right